plugins/source/postgresql/client/list_tables.go

```diff
@@ -76,13 +84,22 @@ func (c *Client) listTables(ctx context.Context) (schema.Tables, error) {
 			table.PkConstraintName = pkName
 		}
 		table.Columns = append(table.Columns, schema.Column{
-			Name: columnName,
-			CreationOptions: schema.ColumnCreationOptions{
-				PrimaryKey: isPrimaryKey,
-				NotNull:    notNull,
-			},
-			Type: c.PgToSchemaType(columnType),
+			Name:       columnName,
+			PrimaryKey: isPrimaryKey,
+			NotNull:    notNull,
+			Unique:     isUnique,
+			Type:       c.PgToSchemaType(columnType),
 		})
 	}
 	return tables, nil
 }
+
+// Add internal columns
+func (c *Client) addInternalColumns(tables []*schema.Table) error {
```

Comment:
```suggestion
func (*Client) addInternalColumns(tables []*schema.Table) error {
```

Reply:
We shouldn't need this method in v4 of the SDK so I removed it

Reply:
yes, but: https://github.com/cloudquery/cloudquery/actions/runs/5334795623/jobs/9667069012?pr=11212#step:7:68